src/pages/thought-leadership/content/index.js

```diff
@@ -67,10 +67,11 @@ export default Vue.component('thought-leadership-content', {
             }, 2000);
           });
         } else {
-          data = await axios({
+          const resp = await axios({
             method: 'get',
-            url: 'http://rackspace.com/api/thought-leadership?_format=json',
+            url: 'api/thought-leadership?_format=json',
```

Comment:
Since this is only loading in `production` mode, which should only occur when embedded in drupal, we should just see it hit the relative endpoint regardless which host we're looking at. Since we make use of the `<base href>` tag in Drupal, it should always pull in the correct data with relative translations (although some tweaks to the view inside Drupal may be warranted).